src/pages/master-teachers-table/master-teachers-table.jsx

```diff
@@ -22,47 +52,114 @@ const MasterTeacherTableView = () => {
     {
       headerTitle: 'Status',
       headerPopover:
-        "<p><strong style='color:#28a745'>Active:</strong> This user is active in the current cycle. They have full access and can log in.</p> <p><strong style='color:#5f758d'>Inactive:</strong> This user is inactive in the current cycle. They cannot log in until an admin user reactivates their account.</p> <p><strong style='color:#17a2b8'>Email Sent:</strong> An email sign up link was sent. They have not set up their account yet.",
+        "<p><strong style='color:#28a745'>Active:</strong> This user is active in the current cycle. They have full access and can log in.</p> <p><strong style='color:#17a2b8'>Email Sent:</strong> An email sign up link was sent. They have not set up their account yet.",
+    },
+    {
+      headerTitle: 'Reset Password',
+      headerPopover: '',
     },
     {
       headerTitle: 'Notes',
       headerPopover: '',
     },
   ];
 
-  useEffect(async () => {
-    const res = await TLPBackend.get(`/teachers`, {
+  // get data to show in the table
+  const parseTableData = data => {
+    const allTeachers = [];
+
+    data.forEach(teacherObj => {
+      const { firstName, lastName, email, phoneNumber, sites, notes, userId, inviteId } =
```

Comment:
It doesn't look like invited MTs who haven't accepted are being displayed

Reply:
nevermind looks like this is because the email invite flow hasn't changed yet for MTs